testing/run-scripts/zork-stats.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Outputs some interesting stats from a zork log.
+
+# TODO: this is preposterously slow
+# TODO: mean negotiation time
+# TODO: mean connection length
+
+set -e
+
+function usage () {
+    echo "$0 [-h] logfile"
+    exit 1
+}
+
+while getopts h? opt; do
+    case $opt in
+        *) usage ;;
+    esac
+done
+shift $((OPTIND-1))
+
+if [ $# -lt 1 ]
+then
+    usage
+fi
+
+IFS="
```

Comment:
Might be worth commenting this as "set field separator to only newlines", as I had to look it up at least.